airflow/settings.py

```diff
@@ -251,16 +251,21 @@ def prepare_classpath():
         sys.path.append(PLUGINS_FOLDER)
 
 
-try:
-    from airflow_local_settings import *  # noqa F403 F401
-    log.info("Loaded airflow_local_settings.")
-except Exception:
-    pass
+def import_local_settings():
+    try:
+        import airflow_local_settings
+        for k, v in airflow_local_settings.__dict__.items():
+            if not k.startswith("__") and not k.endswith("__"):
+                globals()[k] = v
+        log.info("Loaded airflow_local_settings.")
+    except Exception:
```

Comment:
This is just a question, is this kind of exception the only way to do this? Why is it so broad?

Reply:
It is indeed quite broad and could be replaced with `ImportError` or `ModuleNotFoundError`. Thanks for your comment.

Reply:
I have implemented your suggestion. Marking this as resolved.